packages/integration-tests/test/challengeExitTest.js

```diff
@@ -27,7 +27,7 @@ const assert = chai.assert
 const path = require('path')
 const faucetName = path.basename(__filename)
 
-describe('challengeExitTest.js', function () {
+describe.only('challengeExitTest.js', function () {
```

Comment:
just want to double-check, is it intentional to leave `.only` here?

Reply:
good catch! removed